spec/features/providers/allocations/show_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+# Form is currently not yet implemented
+
+feature "Show providers", type: :feature do
+  let(:organisation_show_page) { PageObjects::Page::Providers::Allocations::ShowPage.new }
+end
```

Comment:
Not sure we need this we already have other allocations feature specs. Speaking of which, 

Would you mind moving `features/allocations_request_spec.rb` and `features/allocations_spec.rb to `features/providers/allocations`?

Reply:
I've combined the files and moved them to `features/providers/allocations`